week1/1.5/Notes/Notes/Models/Note.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Note.swift
+//  Notes
+//
+//  Created by Eric Lanza on 7/6/18.
+//  Copyright © 2018 ETLanza. All rights reserved.
+//
+
+import Foundation
+
+class Note: Equatable, Codable {
+   
+    //MARK: - Properties
+    var bodyText: String
```

Comment:
`bodyText` seems like a name inspired by the `UIView` element that represents it and not one that refers to the `Note` in and of itself. `body` as opposed to what? -- nothing because there is nothing providing context to explain what would be non-body.